routes/comments.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+const commentsController = require('../controllers/comments-controller');
+
+router.post('/comment', commentsController.addComment);
+router.get('/comments', commentsController.getAllComments);
+router.get('/comments/:postId', commentsController.getCommentsByPostId);
+router.put('/comment/:id', commentsController.updateComment);
+router.delete('/comment/:id', commentsController.deleteComment);
```

Comment:
(https://github.com/DanaXCohen/web-assignment-1/pull/3/files#r1847140803)
Continuing my comment all your routes won't need the start of /comment/s as they all would be already /comments